rust/osm2lanes/src/lib.rs

```diff
@@ -130,7 +130,7 @@ mod tests {
     #[derive(Deserialize)]
     struct TestCase {
         /// The URL to the OSM way
-        way: String,
+        way_id: u32,
```

Comment:
This should be `i64`, a signed 64-bit. I know negative IDs aren't used in OSM and I can't find a clear answer on the wiki, but I think I haven't seen people use unsigned ints. https://wiki.openstreetmap.org/wiki/Node#Structure

Reply:
You're right. I found that JOSM uses negative identifiers for elements not yet stored in the database.